pyproject.toml

```diff
@@ -32,6 +32,11 @@ dependencies = [
 ]
 dynamic = ["version"]
 
+[project.optional-dependencies]
+all = [
+    "geopandas"
```

Comment:
Should we add `ipython` as well as mentioned in https://www.pygmt.org/v0.7.0/install.html#dependencies?
```suggestion
    "geopandas",
    "ipython"
```

Reply:
Yes, because we also try to import IPython in our codes.